support/Pipelines/Bbh/Inspiral.py

```diff
@@ -114,6 +114,16 @@ def start_inspiral(
     if pipeline_dir and not run_dir and not segments_dir:
         segments_dir = pipeline_dir / "002_Inspiral"
 
+    # Determine resource allocation
+    if (
+        scheduler_kwargs.get("scheduler") is not None
+        and scheduler_kwargs.get("num_procs") is None
+        and scheduler_kwargs.get("num_nodes") is None
+    ):
+        # Just run on 4 nodes for now, because 1 is surely not enough. We can
+        # make this smarter later (e.g. scale with the number of elements).
+        scheduler_kwargs["num_nodes"] = 4
```

Comment:
Can you add this to the help string above so users are aware of it?

Also, I think this is something that should be machine specific? 4 wheeler nodes are not equivalent to 4 Caltech HPC nodes. Not saying to implement this now, 4 seems like an ok default for all our clusters, but for future consideration.

Reply:
Added to the docs. Yes this has to scale with the node size somehow. Maybe we can specify the number of cores (rather than nodes) and make the scheduler select the number of nodes.